tools/torch_quant/torch_quant/graph.py

```diff
@@ -82,6 +85,28 @@ def __init__(self, gm: GraphModule, root: nn.Module,
         self.is_override_module = is_override_module
         self.is_override_qconfig = is_override_qconfig
 
+    @property
```

Comment:
consider using [cached_property](https://docs.python.org/3/library/functools.html#functools.cached_property)?

Reply:
Exactly. But `cached_property` is new in python version 3.8.